androidgcs/google-play-services_lib/project.properties

```diff
@@ -11,5 +11,5 @@
 #proguard.config=${sdk.dir}/tools/proguard/proguard-android.txt:proguard-project.txt
 
 # Project target.
-target=Google Inc.:Google APIs:14
+target=Google Inc.:Google APIs:19
```

Comment:
No, it is actually intentional. You have to compile with a more recent apI to have the BTLE library calls available. However when running on hardware that has an API between 14 and 18, then it will detect this at run time and throw a warning and not make the calls.